src/datepicker/datepicker-input.ts

```diff
@@ -278,6 +279,7 @@ export class NgbInputDatepicker implements OnChanges,
   private _onChange = (_: any) => {};
   private _onTouched = () => {};
   private _validatorChange = () => {};
+  private _destroyCloseHandlers$ = new Subject<void>();
```

Comment:
Could you please move this one ±100 lines higher to other private field group, so it won't get lost here accidentally

Reply:
Yes no problem - I didn't notice that group